src/css/variables.less

```diff
@@ -27,6 +27,9 @@
 @app-status-running-color: #2568F5;
 @app-status-delayed-color: #F04F42;
 @app-status-suspended-color: @table-text-color;
+@task-health-healthy-color: #7aff68;
+@task-health-sick-color: #F04F42;
```

Comment:
From what I gather from the Sketch files healthy should be `00D99C` and sick should be `FD5B77`.